atlas/packages/app/config-overrides.js

```diff
@@ -56,4 +56,20 @@ module.exports = {
     paths.appBuild = path.resolve(__dirname, '..', '..', 'dist')
     return paths
   },
+  jest: function (config) {
+    config.coverageThreshold = {
+      global: {
+        statements: 48,
```

Comment:
Where do these values come from? I've seen you've mentioned that these should be current values - 2, but I don't see that reflected. When I run coverage I get the following values instead 68.66, 45.64, 57.26, 70.52.

Also, more importantly, I'm not sure we should enforce coverage at this point in development. I feel like we can enforce testing some more complex pieces of logic but I don't think we should spend time on writing tests for every component